code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -157,31 +160,25 @@ client/proc/one_click_antag()
 
 	var/list/mob/living/carbon/human/candidates = list()
 	var/mob/living/carbon/human/H = null
-
+	var/antnum = input(owner, "How many cultists you want to create? (0 - 5). Enter 0 to cancel.","Amount:", 0) as num
+	if(!antnum || antnum > 5 || antnum <= 0) // 5 because cultist can really screw balance over if spawned in high amount.

```

Comment:
Remove the upper limit on all of these, please. It's up to the admin to decide what's going to break the round or not.